app/src/main/java/com/nervousfish/nervousfish/activities/MainActivity.java

```diff
@@ -96,6 +73,21 @@ public void onItemClick(final AdapterView<?> parent, final View view, final int
         LOGGER.info("MainActivity created");
     }
 
+    public void bluetoothButtonClicked(final View view) {
```

Comment:
Naming a method in the past tense is very uncommon, maybe change this to `onBluetoothButtonClick` (similar approach for `nfcButtonClicked` and `qrButtonClicked`)